portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,31 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private final ArrayList<String> comments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello stranger! Welcome to my portfolio");
+    // Dummy comments for list
+    comments.add("Test 1");
+    comments.add("Test 2");
+    comments.add("Test 3");
+
+    // Create an object that has the list of comments
+    TestCommentObject commentList = new TestCommentObject(comments);
+
+    // Convert object to JSON
+    String json = convertToJsonUsingGson(commentList);
+    
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJsonUsingGson(TestCommentObject commentList) {
+    Gson gson = new Gson();
+    String json = gson.toJson(commentList);
+    return json;
```

Comment:
Consider condensing this to avoid initializing an extra variable:
```suggestion
    return gson.toJson(commentList);
```

Reply:
Done!